backend/tests/test_calculator.py

```diff
@@ -1,137 +1,25 @@
 import server.calculator as calculator
+import json
 
 
 def test_calculate_fixed_rate():
-    assert (
-        calculator.calculate_fixed_rate(
-            credit_amount=100000, loan_term=10, interest_rate=3
-        )
-        == 965.61
-    )
+    assert calculator.calculate_fixed_rate(
+        credit_amount=100000, loan_term=10, interest_rate=3
+    ) == {
+        "monthly_payment": [965.61] * 120,
+        "total_cost": 115873.2,
+        "total_interest": 15873.2,
+    }
 
 
 def test_calculate_descending_rate():
+    f = open("tests/output/test_calculator_output.json")
+    data = json.load(f)
+    f.close()
```

Comment:
Just use context manager in here 

```suggestion
    with open("tests/output/test_calculator_output.json") as fh:
        data = json.load(fh)
```